.circleci/config.yml

```diff
@@ -47,6 +47,7 @@ jobs:
 
    coverage:
      executor: ubuntu-build
+     resource_class: 2xlarge
```

Comment:
Should this be in a separate PR from the build fix?

Reply:
Sigh, seems we don't have 2xlarge available in our circle account. But I think we can do this in same PR.